apps/frontend/src/app/pages/person/gpt-chat/gpt-chat.component.ts

```diff
@@ -110,14 +131,18 @@ export class GptChatComponent implements OnInit {
           this.isStartingTheChat = false;
         },
         error: ({ error }: { error: HttpError }) => {
```

Comment:
Since the error handling functions are almost identical, you could refactor to a separated function